v0/destinations/marketo/util.js

```diff
@@ -91,14 +104,13 @@ const marketoResponseHandler = (destResponse, sourceMessage, stage) => {
   }
   // non 2xx failure
   throw new ErrorBuilder()
-    .setStatus(status)
+    .setStatus(400)
     .setMessage(`Error occured ${sourceMessage}`)
     .setDestinationResponse(destResponse)
     .setStatTags({
       destType: DESTINATION,
       stage,
-      scope: TRANSFORMER_METRIC.MEASUREMENT_TYPE.API.SCOPE,
-      meta: getDynamicMeta(status)
+      scope: TRANSFORMER_METRIC.MEASUREMENT_TYPE.EXCEPTION.SCOPE
```

Comment:
Why is it an exception scope?

Reply:
Later on it would be difficult to debug if we set API scope

Reply:
Can you please clarify?

Reply:
This is done for two things
- To indicate that the response sent is one of the unhandled status-code cases
- To get ourselves notified(as `exception` is highest priority) saying "this is an unhandled case, please handle it"

Let me know if it makes sense

Reply:
We are misusing the scopes. If that's a workaround for now, leave a comment to refactor later.

Reply:
I don't think we are mis-using the scopes at all. It calls out for an unhandled error case in terms of response from destination. But if you feel strongly about me mis-using the scope, will change it

Reply:
The changes have been, can you check it and let me know if the scope is properly set ?